src/main/java/org/mockbukkit/mockbukkit/block/data/DecoratedPotDataMock.java

```diff
@@ -0,0 +1,71 @@
+package org.mockbukkit.mockbukkit.block.data;
+
+import com.google.common.base.Preconditions;
+import org.bukkit.Material;
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.data.type.DecoratedPot;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Set;
+
+public class DecoratedPotDataMock extends BlockDataMock implements DecoratedPot
+{
+
+
+	/**
+	 * Constructs a new {@link DecoratedPotDataMock} for the provided {@link Material}.
+	 *
+	 * @param material The material this data is for.
+	 */
+	public DecoratedPotDataMock()
+	{
+		super(Material.DECORATED_POT);
+		super.set(BlockDataKey.CRACKED, false);
+		super.set(BlockDataKey.FACING, BlockFace.NORTH);
+		super.set(BlockDataKey.WATERLOGGED, false);
```

Comment:
Should we use the "super" key word here? Maybe the "this" or none would fit better. Because it may be overwritten later.

Reply:
Good point, I was mostly just trying to be clear where the method comes from, but that is not that relevant in this context anyhow; will change